### PR TITLE
Single movie page view

### PR DESCRIPTION
**Pull request by MikeAdamczyk**

### CODE REVIEW

File: src/features/movies/SingleMoviePage/index.js [hunk 1]
```diff
@@ -1,17 +1,48 @@
...
+                                image={result.profile_path}
+                            />
+                        ))}
+                    </ListContainer>
+                </Wrapper>
+
+            </DetailsContainer>
+        </>
+    )
+};
```
[the remark this answers is not in the document]
Comment (damdlani) [reply src/features/movies/SingleMoviePage/index.js:48]: + you're missing a ListContainer with Cast

File: src/theme.js [hunk 2]
```diff
@@ -17,7 +17,7 @@ export const theme = {
         large: "1384px",
         big: "1036px",
         medium: "800px",
-        mobile: "767px", 
+        mobile: "767px",
```
Comment (damdlani) [inline src/theme.js:20]: I changed that one to 768px, so iPad could have a mobile view, more pleasable

**Review by damdlani — APPROVED** [review]
Well done, be careful with conflicts while merging, as we were partly working on the same thing. In main list tile already work as link, but just to one page. Both, in mine and yours changes we edited singlePage - please leave your version, but with my Tile left, as I mentioned in one of the comments.

File: src/features/movies/SingleMoviePage/index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (MikeAdamczyk) [reply src/features/movies/SingleMoviePage/index.js:48]: yes, Cast is needed as well :)

File: src/common/Signs/styled.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+    @keyframes spin {
+        from {
+            transform:rotate(0deg);
+        }
+        to {
+            transform:rotate(360deg);
+        }
+    }
+`;
```
[the remark this answers is not in the document]
Comment (MikeAdamczyk) [reply src/common/Signs/styled.js:19]: yes, it works :) I have checked it :)

File: src/theme.js (hunk 2 above)
Comment (MikeAdamczyk) [reply src/theme.js:20]: Great :) 👍

**Pull request merged**